core/controllers/blog_admin.py

```diff
@@ -106,33 +116,52 @@ def get(self) -> None:
     @acl_decorators.can_access_blog_admin_page
     def post(self) -> None:
         """Handles POST requests."""
-        result = {}
-        if self.normalized_payload.get(
-                'action') == 'save_config_properties':
-            new_config_property_values = self.normalized_payload.get(
-                'new_config_property_values')
+        assert self.user_id is not None
+        payload_data = cast(
+            BlogAdminHandlerNormalizedPayloadDict, self.normalized_payload
+        )
```

Comment:
Added PTAL!

Reply:
Why wasn't this caught by the linter?

Reply:
Because previously `core/controllers` is not removed from the exclude list https://github.com/oppia/oppia/blob/c7ea503b0c545951eb7d8d977e5548d3b53cca37/scripts/linters/pylint_extensions.py#L55

My main motive to keep `core/controllers` in exclude list is that in some files we are using Any type and those files are not annotated yet. But yeah, this thing is fixed by pointing `EXCLUDED_TYPE_COMMENT_DIRECTORIES` to `run_mypy_checks.NOT_FULLY_COVERED_FILES`. So, whenever a file is removed from MyPy deny list then linters are automatically imposed on that file.

Reply:
Great! Thanks a lot. I think it will save your time as you don't have to go over these files later just to write the comments.